tests/data/metadata.yml

```diff
@@ -1,21 +1,23 @@
 ---
 # manual_metadata
+# Please find the BIDS specification here: https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html
+# Please go through the specification and fille the fields below according to the data types given in the specification.
 
-# Required, fill according to BIDS specification:
+# REQUIRED, fill according to BIDS specification (!the converter will not run successfully if you leave this empty!):
 SampleCoordinateUnits: pixel
```

Comment:
I meant here the 'pixel'

Reply:
Also the values of size are in meter

Reply:
> I meant here the 'pixel'

I know, like I said, I put !!str around it to make it clear that it should be a string but as soon as the CI starts here it disappears.

Reply:
> Also the values of size are in meter

Which size? ScreenSize?

Reply:
Check the doc I think all size

Reply:
Ok, the glossary says it's meter. So we need to change the example on the doc, there it is in cm. I open an issue for it.

Reply:
Changed it in the files here, too